khal/khalendar/event.py

```diff
@@ -377,30 +377,44 @@ def _create_calendar() -> icalendar.Calendar:
         )
         return calendar
 
+
     @property
     def raw(self) -> str:
-        """Creates a VCALENDAR containing VTIMEZONEs
-        """
+        """Creates a VCALENDAR containing VTIMEZONEs."""
         calendar = self._create_calendar()
         tzs = []
```

Comment:
**non-blocking nitpick**: I would use a set here, so we can just insert new entries without having any duplicates (inserting an existing element into a set is a no-op):

```suggestion
        tzs = {}
```